### DISCUSSION TITLE
Run Raven 6.0 on Mac M2

### DISCUSSION THREAD

**Discussion by SbiCA** [ravendb/ravendb#17452] [Q&A] [answered]
Hey @ppekrol 

I just downloaded 6.0 and tried to run it on my M2 and get the following error.

mac OS Sonoma 14.0 (23A344) is that a know issue ?

![image](https://github.com/ravendb/ravendb/assets/11393585/f9e136ee-1986-4fbc-a33c-38ac7dbb71bd)

**Comment by SbiCA** [comment]
Fyi @ppekrol  as a workaround, docker image runs fine 

``` bash
docker run -p 8082:8080 -d --name raven6 ravendb/ravendb:6.0.0-ubuntu.22.04-arm64v8
```

**Reply by ppekrol** [reply, marked as answer]
Hi @SbiCA 

You are correct, our trigger for macOS notarization didn't fire. We just ran the process manually for 6.0.0, it should be fine now, but you need to re-download the files.

**Reply by SbiCA** [reply]
@ppekrol  I can confirm now all works 🎉

**Comment by djordjedjukic** [comment]
@ppekrol 
I have the same problem, the version is **6.0.103** (the latest one). 
Can you check if it is the same issue?

**Reply by ppekrol** [reply]
Fixed, thanks @djordjedjukic
